src/library/library.cpp

```diff
@@ -223,6 +223,26 @@ void Library::stopPendingTasks() {
     }
 }
 
+void Library::bindSearchboxWidget(WSearchLineEdit* pSearchboxWidget) {
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::search,
+            this,
+            &Library::search);
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::disableSearch,
+            this,
+            &Library::disableSearch);
+    connect(pSearchboxWidget,
+            &WSearchLineEdit::restoreSearch,
+            this,
+            &Library::restoreSearch);
+    connect(this,
+            &Library::setTrackTableFont,
```

Comment:
have we settled on this line-wrap style for connections?  It seems excessive, you could do it in two readable lines:

```
connect(pSearchboxWidget, &WSearchLineEdit::restoreSearch,
        this, &Library::restoreSearch);
```

Reply:
I tried to stick to the scheme used for recent changes in library.cpp
I don't mind either way..

Reply:
It's a perfectly valid choice to spread arguments on separate lines according to our coding guidelines. It is also preserved by clang-format.